### PR TITLE
Added backup goal to makefile

### PR DESCRIPTION
**Pull request by duozersk**
Added backup goal to makefile

### CODE REVIEW

**Comment by gaydabura** [comment]
What is the real reason of this goal? As we build site from default content, i mean all data comes from default content , and after some work is done usually export config and content, so i dont see any reason to make backup and add this goal to initial image here

**Comment by duozersk** [comment]
So let's get this in and create followup to develop import goal - sounds good? I would suggest to call it "restore"... like export/import; backup/restore

File: Makefile [hunk 1]
```diff
@@ -157,6 +157,15 @@ dev:
 	$(call php, drush en devel devel_generate webform_devel kint -y)
 	$(call php, drush pm-uninstall dynamic_page_cache page_cache -y)
 
+backup_name = $(COMPOSE_PROJECT_NAME).tar.gz
```
Comment (andypost) [inline Makefile]: It needs current date & time- no reason to have backup with name only

File: Makefile [hunk 2]
```diff
@@ -157,6 +157,15 @@ dev:
...
+mysql_dump_name = $(COMPOSE_PROJECT_NAME).sql
+files_dir = web/sites/default/files
```
Comment (andypost) [inline Makefile]: this king of settings better to have at top of the file at least

**Comment by gaydabura** [comment]
This PR is not needed anymore

**Pull request closed without merging**